airbyte-integrations/connectors/source-plausible/integration_tests/invalid_config.json

```diff
@@ -2,4 +2,4 @@
   "api_key": "<invalid_key>",
   "site_id": "https://airbyte.com",
   "start_date": "20220101"
-}
+}
```

Comment:
Please remove the file from this PR as it is unrelated to the LinkedIn Ads source connector.